k8s/aat/common/ccpay/bar-app.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bar-app
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bar-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bar-api
+    version: 0.2.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/bar/api:latest
+      ingressHost: bar-api-aat.service.core-compute-aat.internal
```

Comment:
https://github.com/hmcts/bar-apps/pull/281/files#diff-3f5b26fc2d25bf67914541be23568b07R4 This should handle it.
```suggestion
```